### ISSUE TITLE
Description of loading JQuery from CDN needs improvement

### ISSUE THREAD

**Issue by ToonTalk** [ox-it/LiveData_htmlwidgets#39]
Regarding

> In the explicit case of jquery, the following code can be used to load the local resource if the CDN is unavailable:
> 
> <script src="http://ajax.aspnetcdn.com/ajax/jquery/jquery-2.0.0.min.js"></script>
> 
> <script>
> if (typeof jQuery == "undefined") {
>     document.write(unescape("%3Cscript src="/js/jquery-2.0.0.min.js" type="text/javascript"%3E%3C/script%3E"));
> }
> </script>

Need to explain why one would want to do this. And if you give a specific version number then I'm not sure what the advantage is over using the local copy (other than it might be cached and hence save the downloading cost (which is trivial in this case)). Maybe make this example consistent with the jquery.min.js example that precedes it.

Also your script will download it from /js/jquery instead of shared/jquery.min.js

**Comment by charliejhadley** [comment]
Ken made a good point in meatspace that any shiny app will be pulling from the US and not using a closer CDN. A hat tip to this would be worth making, and if providing a visualisation service with a privately hosted Shiny Server (Pro), how might one attempt to resolve this?

**Comment by charliejhadley** [comment]
Ken also advised in meatspace that in JS it is best to use === and not == for equivalence testing
